async-http-client-backend/cats/src/main/scala/sttp/client3/asynchttpclient/cats/AsyncHttpClientCatsBackend.scala

```diff
@@ -38,7 +37,7 @@ class AsyncHttpClientCatsBackend[F[_]: Concurrent: ContextShift] private (
   override val streams: NoStreams = NoStreams
 
   override def send[T, R >: Any with sttp.capabilities.Effect[F]](r: Request[T, R]): F[Response[T]] = {
-    super.send(r).guarantee(implicitly[ContextShift[F]].shift)
```

Comment:
this whole override can be removed now, since we don't do anything special here anymore

Reply:
👍 

@SystemFw Just for confirmation: by looking at the [release notes](https://github.com/typelevel/cats-effect/releases/tag/v3.0.0-M1), I understand that we don't need the `ContextShift[F]].shift` to shift back to the appropriate thread pool. Can you confirm? Do we still need to provide the `Async[F].cede` hint somehow?

Reply:
you don't need to do anything, `async` will shift back on its own internally

Reply:
Thanks for the confirmation.

Reply:
https://github.com/softwaremill/sttp/pull/738/commits/6b78c3c5eb0a33b17ff8ba79743a464071da5f27